psi4/driver/procrouting/sapt/sapt_jk_terms.py

```diff
@@ -97,8 +97,8 @@ def build_sapt_jk_cache(wfn_A, wfn_B, jk, do_print=True):
 
     # K_O J/K
     C_O_A = core.triplet(
-        cache["D_B"], cache["S"], cache["Cocc_A"], False, False, False)
-    jk.C_left_add(C_O_A)
+        cache["D_B"], cache["S"], cache["Cocc_A"], False, False, False) 
+    jk.C_left_add(C_O_A) 
```

Comment:
you'll want to run yapf over py files and clang-format over C++ files so that don't get false changes like these.

Reply:
Is it an automatic formatter? I didn't know this before...